### PR TITLE
fix: OCI registry integration UI bugs fixes and requirement updates

### PR DESCRIPTION
**Pull request by Ash-exp**
# Description

fixed UI bugs and updates for OCI registry integration in Global config.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
- [x] UI reviewed

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -432,6 +432,14 @@ export const RegistryStorageType = {
     CONTAINER: 'CONTAINER',
     OCI_PRIVATE: 'OCI_PRIVATE',
 }
+
+export const REGISTRY_TITLE_DESCRIPTION_CONTENT = {
```
Comment (vivek-devtron) [inline src/config/constants.ts:436]: format this file

Comment (Ash-exp) [reply src/config/constants.ts:436]: formatted 👍

File: src/components/dockerRegistry/Docker.tsx [hunk 2]
```diff
@@ -165,7 +187,7 @@ function CollapsedList({
     return (
         <article className={`collapsed-list collapsed-list--docker collapsed-list--${id ? 'update' : 'create dashed'}`}>
             <List
-                dataTestId={id || 'Add'}
+                dataTestId={id || 'add-registry-button'}
```
Comment (vivek-devtron) [inline src/components/dockerRegistry/Docker.tsx:190]: is it expected?

File: src/components/dockerRegistry/Docker.tsx [hunk 3]
```diff
@@ -183,7 +205,7 @@ function CollapsedList({
                 <div className="flex left">
                     <List.Title
                         style={{ color: !id && !collapsed ? 'var(--N900)' : '' }}
-                        title={id || 'Add Container Registry'}
```
Comment (vivek-devtron) [inline src/components/dockerRegistry/Docker.tsx:186]: is it expected?

File: src/components/dockerRegistry/Docker.tsx [hunk 4]
```diff
@@ -487,6 +509,14 @@ function DockerForm({
         }
     }
 
+    const handleDefaultChange = (e) => { 
+        if (isDefault) {
+            toast.success('Please mark another as default.')
+            return
+        }
+        toggleDefault((t) => !t)
```
Comment (vivek-devtron) [inline src/components/dockerRegistry/Docker.tsx:517]: we can write it as `toggleDefault(not)`

Comment (Ash-exp) [reply src/components/dockerRegistry/Docker.tsx]: Incorporated 👍

File: src/components/dockerRegistry/Docker.tsx (hunk 3 above)
Comment (Ash-exp) [reply src/components/dockerRegistry/Docker.tsx:186]: expected as now we will also configure OCI registry here

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**